src/common/SPELLS/bfa/essences/essences.js

```diff
@@ -203,6 +203,11 @@ export default {
     name: 'Lucid Dreams',
     icon: 'inv_radientazeritematrix',
   },
+  LUCID_DREAMS_MINOR_RESOURCE_REFUND_WARRIOR: {
```

Comment:
Guardian Druids also use rage ( and this exact spellId). Rename it to `LUCID_DREAMS_MINOR_RESOURCE_REFUND_RAGE` and i'll merge it